kernel/src/fs/ext2/inode.rs

```diff
@@ -2230,6 +2232,21 @@ impl InodeDesc {
         }
         size.div_ceil(BLOCK_SIZE) as Ext2Bid
     }
+
+    fn sectors_to_blocks(sector_count: u32) -> Ext2Bid {
```

Comment:
Both `sectors_to_blocks` and `blocks_to_sectors` have no inherent connection with `InodeDesc`. They should be made static utility functions, rather than methods.